app/src/main/kotlin/no/nav/omsorgsdager/aleneomsorg/AleneOmsorgBehandling.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.omsorgsdager.aleneomsorg
+
+import no.nav.omsorgsdager.K9BehandlingId
+import no.nav.omsorgsdager.K9Saksnummer
+import no.nav.omsorgsdager.behandling.BehandlingStatus
+import no.nav.omsorgsdager.behandling.EksisterendeBehandling
+import no.nav.omsorgsdager.parter.Barn
+import no.nav.omsorgsdager.parter.Søker
+import no.nav.omsorgsdager.tid.Periode
+import java.time.ZonedDateTime
+
+internal data class AleneOmsorgBehandling (
+    override val k9Saksnummer: K9Saksnummer,
+    override val k9behandlingId: K9BehandlingId,
+    override val tidspunkt: ZonedDateTime,
+    override val periode: Periode,
+    internal val barn: Barn,
+    internal val søker: Søker,
+    override val status: BehandlingStatus) : EksisterendeBehandling {
+    override val enPer: K9Saksnummer = k9Saksnummer
```

Comment:
Tror denne bør være som for kronisk-sykt-barn? `enPer: Barn = barn`

Reply:
hvis vi gjør det, er det slik at bare en person som kan få godkjent alene omsorg for barnet i en viss periode?
Bør forsåvidt være ok

Reply:
Nja, tror ikke det har noe funksjonell forskjell om man får nytt saksnummer per person+barn-kombinasjon.

Ble vel laget slik for kronisk-sykt-barn da det ikke var sikkert om det ble et nytt saksnummer for nytt barn, eller om en søker hadde et saksnummer og under det kunne gjelde forskjellige barn.

Om man for alene-om får nytt saksnummer for hvert barn så er det hipp som happ

Reply:
Ok, jeg bytter over